ansible_collections/arista/avd/docs/release-notes/3.x.x.md

```diff
@@ -8,6 +8,20 @@
 
 ## Release 3.8.0
 
+!!! info "Variable precedence changed in eos_cli_config_gen"
+
+    In cases where `eos_cli_config_gen` is invoked in a separate playbook, variables are imported from "structured_config" files. These variables are now imported as regular facts with `host_facts` precedence instead of the special `include_vars` precedence.
```

Comment:
```suggestion
    In cases where `eos_cli_config_gen` is invoked in a separate playbook, variables are imported from "structured_config" files. These variables are now imported as regular facts with `host_facts` precedence instead of the particular `include_vars` precedence.
```